packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -997,6 +1010,10 @@ export class StatsAnalyzer extends EventsScope {
       this.statsResults[mediaType][sendrecvType].headerBytesSent = result.headerBytesSent;
       this.statsResults[mediaType][sendrecvType].retransmittedBytesSent =
         result.retransmittedBytesSent;
+      this.statsResults[mediaType][sendrecvType].isRequested =
+        result.isRequested ||
+        (result.lastRequestedUpdateTimestamp &&
+          performance.timeOrigin + performance.now() - result.lastRequestedUpdateTimestamp < 60000);
```

Comment:
There is a var for timeout `MQA_INTERVAL` Suggest using it here and other code parts with timestamps